lib/monitoring/cloudwatch.js

```diff
@@ -1,14 +1,14 @@
 const AWS = require('aws-sdk');
 const fs = require('fs');
 
-const AWS_CONFIG_FILENAME = './config/aws-conf.json';
+const AWS_CONFIG_FILENAME = './aws-conf.json';
```

Comment:
Why change the filepath to not be in a config directory?

Reply:
No good reason. I'll change that.